crates/ruff_linter/resources/test/fixtures/ruff/suppressions.py

```diff
@@ -0,0 +1,45 @@
+def f():
+    # These should both be ignored by the range suppression.
+    # ruff: disable[E741, F841]
+    I = 1
+    # ruff: enable[E741, F841]
+
+
+def f():
+    # These should both be ignored by the range suppression.
+    # ruff:disable[E741,F841]
+    I = 1
+    # ruff:enable[E741,F841]
+
+
+def f():
+    # These should both be ignored by the range suppression.
+    # ruff: disable[E741]
+    # ruff: disable[F841]
+    I = 1
+    # ruff: enable[E741]
+    # ruff: enable[F841]
```

Comment:
I think it would be good to add/move some of the more advanced test cases involving blocks and make them integration tests instead.